loader/loader.go

```diff
@@ -248,26 +256,31 @@ type ProcessingResources struct {
 // be retrieved from the cluster, an error will be returned.
 func (l *loader) GetProcessingResources(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*ProcessingResources, error) {
 	var err error
+
 	resources := &ProcessingResources{}
 
 	resources.ReleasePlan, err = l.GetReleasePlan(ctx, cli, release)
 	if err != nil {
 		return resources, err
 	}
 
-	resources.ReleasePlanAdmission, err = l.GetActiveReleasePlanAdmissionFromRelease(ctx, cli, release)
-	if err != nil {
-		return resources, err
-	}
+	// Will only load the ReleasePlanAdmission, EnterpriseContractConfigMap and EnterpriseContractPolicy if ReleasePlan.Spec.Target
```

Comment:
This function gets 5 resources. Conditionally ignoring 3 of them seems weird to me. I think it may be better to rename this to getmanagedprocessingresources and then create one for the tenant pipelines

Reply:
Honestly, for just two resources I am not even sure we need a composite function

Reply:
Good point. I will think what might be better - and handy.

Reply:
It made sense before and it still makes sense when you get the processing resources for the managed Pipeline. I think the problem here is that we are reusing functions that were used for the managed Pipeline for both of them now. Functions are cheap. We could define new ones. And in the case of the tenant Pipeline, no need for a new composite function.

Reply:
Yeah, I am in favor of not modifying GetProcessingResources. I think you have already did this based on the adapter code, but the function is still modified in loader.go

Reply:
I removed the changes here as well.